charts/cromwell/templates/serviceMonitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cromwell-jvm-monitor
+  labels: 
+{{ include "cromwell.labels" . | indent 4 }}
+spec:
+  jobLabel: cromwell-jvm
+  selector:
+    matchLabels:
+      prometheus.io/scrape: "true"
+  namespaceSelector:
+    matchNames:
+    - terra-dev
```

Comment:
Do we only want this for dev?

Reply:
Oh I'll take this out. This was from before updated the prometheus config to support a more general pattern. Not needed anymore.

Reply:
Just tested with the namespace selector removed and confirmed everything still works as intended